tests/e2e/scale/test_scale_osds_fill_75%_reboot_workers.py

```diff
@@ -86,7 +86,7 @@ def test_scale_osds_reboot_nodes(self, interface, project_factory, multi_pvc_fac
             selector='app=rook-ceph-osd',
             resource_count=count * 3
         )
-        assert ceph_health_check(), "New OSDs failed to reach running state"
+        assert ceph_health_check(delay=120, tries=50), "New OSDs failed to reach running state"
```

Comment:
Just out of curiosity, Why do we need 50 tries here, is 10, 15 tries not enough ?

Reply:
After filling cluster with number of data and add capacity to the cluster, it requires more time to balance the data and pg reaches to active and clean state.

Reply:
Do we have a function to check if there are any on-going rebalance operations? If not, having one would be useful in making such timeout decisions.

Reply:
I don't think we have a function to check this.  It can be addressed in a separate PR.

Reply:
Sure, maybe let's raise an issue to track this.